### ISSUE TITLE
pnpm install error

### ISSUE THREAD

**Issue by toledoroy** [turbo-eth/turbo-eth#3]
pnpm install error 
`   husky - .git can't be found (see https://typicode.github.io/husky/#/?id=custom-directory)`

<img width="656" alt="image" src="https://user-images.githubusercontent.com/5618090/174947274-2de7db0f-3f9b-40e8-93cc-5a086678f3e6.png">

Happened on windows 11
Also happened when trying to deploy on Vercel

**Comment by kamescg** [comment]
hi @toledoroy. thanks for opening up the first issue.

the husky warnings are leftover from the individual repos. the husky commands should probably be moved to the top level folder, but just haven't had a chance yet. that being said shouldn't prevent installing and running the build system. 

 this coming week going to clean-up up the repo to remove these types of warnings.

**Comment by toledoroy** [comment]
That's great. Thanks! 
It's a great repo but we had numerous issues when trying to deploy this to a vercel server. Ethers, husked, lint, and other things broke the build (works locally though) 
I can share with you the address for the fork that eventually worked if you want.

**Comment by wottpal** [comment]
I don't know if this is related (probably not). But I also have errors when installing dependencies:

```
➜  turbo-eth git:(main) pnpm i    
Scope: all 6 workspace projects
Lockfile is up-to-date, resolution step is skipped
Packages: +2264
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
Packages are hard linked from the content-addressable store to the virtual store.
  Content-addressable store is at: /Users/wottpal/Library/pnpm/store/v3
  Virtual store is at:             node_modules/.pnpm
Downloading registry.npmjs.org/solc/0.8.13: 6.01 MB/6.01 MB, done
Downloading registry.npmjs.org/ganache-cli/6.12.2: 5.8 MB/5.8 MB, done
Downloading registry.npmjs.org/solidity-coverage/0.7.18: 8.16 MB/8.16 MB, done
Downloading registry.npmjs.org/typescript/3.9.10: 9.58 MB/9.58 MB, done
Progress: resolved 2264, reused 1478, downloaded 768, added 2264, done
node_modules/.pnpm/core-js-pure@3.23.1/node_modules/core-js-pure: Running postinstall script...
node_modules/.pnpm/core-js-pure@3.23.1/node_modules/core-js-pure: Running postinstall script, done in 41mspm/web3-shh@1.5.3/node_modules/web3-shh: Running postinstall script, done in 12ms
node_modules/.pnpm/web3@1.2.11/node_modules/web3: Running postinstall script, done in 43ms
node_modules/.pnpm/web3@1.5.3/node_modules/web3: Running postinstall script, done in 8ms
node_modules/.pnpm/esbuild@0.14.46/node_modules/esbuild: Running postinstall script, done node_modules/.pnpm/turbo@1.2.16/node_modules/turbo: Running postinstall script, done in 104ms

dependencies:
+ @babel/core 7.18.5
+ @coinbase/wallet-sdk 3.3.0
+ eslint-config-custom 0.0.0
+ react 18.2.0
+ react-dom 18.2.0

devDependencies:
+ prettier 2.7.1
+ turbo 1.2.16

packages/core-wagmi prepare$ tsdx build
│ @rollup/plugin-replace: 'preventAssignment' currently defaults to false. It is recomme
│ @rollup/plugin-replace: 'preventAssignment' currently defaults to false. It is recomme
│ ⠙ Creating entry file
│ ✓ Creating entry file 542 ms
│ ⠙ Building modules
│ ⠹ Building modules
│ ⠸ Building modules
└─ Running...
packages/erc20-wagmi prepare$ tsdx build
│ @rollup/plugin-replace: 'preventAssignment' currently defaults to false. It is recomme
│ @rollup/plugin-replace: 'preventAssignment' currently defaults to false. It is recomme
│ ⠙ Creating entry file
│ ✓ Creating entry file 595 ms
│ ⠙ Building modules
│ ⠹ Building modules
│ ⠸ Building modules
│ (typescript) Error: /Users/wottpal/Developer/playground/turbo-eth/packages/erc20-wagmi
│ Error: /Users/wottpal/Developer/playground/turbo-eth/packages/erc20-wagmi/src/ERC20App
│     at error (/Users/wottpal/Developer/playground/turbo-eth/node_modules/.pnpm/rollup@
│     at throwPluginError (/Users/wottpal/Developer/playground/turbo-eth/node_modules/.p
│     at Object.error (/Users/wottpal/Developer/playground/turbo-eth/node_modules/.pnpm/
│     at Object.error (/Users/wottpal/Developer/playground/turbo-eth/node_modules/.pnpm/
│     at RollupContext.error (/Users/wottpal/Developer/playground/turbo-eth/node_modules
│     at /Users/wottpal/Developer/playground/turbo-eth/node_modules/.pnpm/rollup-plugin-
│     at arrayEach (/Users/wottpal/Developer/playground/turbo-eth/node_modules/.pnpm/rol
│     at Function.forEach (/Users/wottpal/Developer/playground/turbo-eth/node_modules/.p
│     at printDiagnostics (/Users/wottpal/Developer/playground/turbo-eth/node_modules/.p
│     at Object.transform (/Users/wottpal/Developer/playground/turbo-eth/node_modules/.p
│ 
└─ Failed in 2.6s
packages/erc721-wagmi prepare$ tsdx build
[1 lines collapsed]
│ @rollup/plugin-replace: 'preventAssignment' currently defaults to false. It is recomme
│ ⠙ Creating entry file
│ ✓ Creating entry file 495 ms
│ ⠙ Building modules
│ ⠹ Building modules
│ Generated an empty chunk: "index"
│ Generated an empty chunk: "index"
│ Generated an empty chunk: "index"
│ ⠸ Building modules
│ ✓ Building modules 1.1 secs
└─ Running...
 ELIFECYCLE  Command failed with exit code 1.
```

Those are my installed versions:
```
➜  turbo-eth git:(main) pnpm --version
7.5.0
➜  turbo-eth git:(main) npm --version
8.13.1
➜  turbo-eth git:(main) node --version                      
v16.15.1
```

Let me know if I should open a new issue with that, @toledoroy :)

**Comment by wottpal** [comment]
I debugged it a bit more. It seems like `ethers` is missing as a dependency in `packages/erc20-wagmi`:

```
(typescript) Error: /Users/wottpal/Developer/playground/turbo-eth/packages/erc20-wagmi/src/ERC20Balance.tsx(4,23): semantic error TS2307: Cannot find module 'ethers' or its corresponding type declarations.
Error: /Users/wottpal/Developer/playground/turbo-eth/packages/erc20-wagmi/src/ERC20Balance.tsx(4,23): semantic error TS2307: Cannot find module 'ethers' or its corresponding type declarations.
```

After adding it via `cd packages/erc20-wagmi && pnpm add -D ethers` it runs through ;)

**Comment by kamescg** [comment]
Fixed in #8
